gui/wxpython/rlisetup/wizard.py

```diff
@@ -1731,15 +1728,13 @@ def SampleFrameChanged(self, region):
             self.parent.wizard.ShowPage(self.parent.summarypage)
         else:
             self.title.SetLabel(
-                _(
-                    "Draw Sampling "
-                    + drawtype
-                    + " "
-                    + str(self.regioncount)
-                    + " of "
-                    + str(self.numregions)
+                _("Draw sampling {draw_type} {count} of {num_regions}").format(
+                    draw_type=drawtype,
+                    count=self.regioncount,
+                    num_regions=self.numregions,
                 )
             )
+
```

Comment:
```suggestion
```

Reply:
Hello! I removed the line and committed changes. Hope, now it's fine)